src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -60,7 +86,7 @@
         max-height: 0px;
         overflow: hidden;
         right: 0;
-        top: 100%;
+        top: 112%;
```

Comment:
with this change you can't hover from the button to the menu, since the space between it will count as un-hovering the button, hiding the menu (in the main branch you can currently hover the menu without clicking the button)

Reply:
Ok, found a fix now, it is working fine now and we kept the spacing.